finalPrep/modZeroHero.js

```diff
@@ -1,41 +1,60 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "Deku";
+var specialAbility = "OFA";
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
-
+var greeting, catchphrase;
+greeting = "Hello, I am " + heroName;
+catchphrase = specialAbility + " is my power, it stands for One For All. I inherited it from my master.";
```

Comment:
They wanted you to use interpolation here. So something like `${specialAbility} is my power, it stands for One For All. I inherited it from my master.` You used concatenation for both.